install.sh

```diff
@@ -770,10 +713,12 @@ install_buildah(){
       ;;
   esac
 
+  is_command_exists usermod || abort "usermod not found" 
+   
   # Configure user namespaces
   if ! grep -q "$(get_user)" /etc/subuid; then
```

Comment:
Maybe we should do this instead?
```bash
if grep /etc/subuid; then
  # add entry to /etc/subuid
fi

if grep /etc/subgid; then
  # add entry to /etc/subgid
fi
```

I guess there might be cases where entries in two of these files are not identical.